sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/implementation/KeyVaultCredentialPolicy.java

```diff
@@ -67,16 +72,20 @@ private static Map<String, String> extractChallengeAttributes(String authenticat
             return Collections.emptyMap();
         }
 
-        authenticateHeader =
-            authenticateHeader.toLowerCase(Locale.ROOT).replace(authChallengePrefix.toLowerCase(Locale.ROOT), "");
-
-        String[] attributes = authenticateHeader.split(", ");
+        String[] attributes = authenticateHeader.substring(authChallengePrefix.length()).split(", ");
         Map<String, String> attributeMap = new HashMap<>();
 
         for (String pair : attributes) {
-            String[] keyValue = pair.split("=");
+            // This is ugly, but we need to trim here because currently the 'claims' attribute comes after two spaces.
+            pair = pair.trim();
 
-            attributeMap.put(keyValue[0].replaceAll("\"", ""), keyValue[1].replaceAll("\"", ""));
+            if (pair.startsWith("claims=")) {
+                attributeMap.put("claims", pair.substring("claims=".length()).replaceAll("\"", ""));
+            } else {
+                String[] keyValue = pair.split("=");
```

Comment:
nit: Use `split("=", 2)`, right now if there are additional `=` in the value portion of the key-value pair (which can happen with base64 values as they sometimes need trailing `=`) this will result in more than two values and will likely break something as trailing `=` would be dropped.